src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.yml

```diff
@@ -293,23 +294,23 @@ repository:
   #>>>>> @addTagHelper, @removeTagHelper and @tagHelperPrefix <<<<<
 
   addTagHelper-directive:
-    name: meta.directive.addTagHelper.razor
+    name: 'meta.directive.addTagHelper.razor'
```

Comment:
Is this just a YAML lint thing?

Reply:
Hmm, yaml linter, interesting probably something we should add. That being said these changes are more about making all the bits consistent 😄. I did it by hand.